lib/features/thread/presentation/thread_controller.dart

```diff
@@ -100,17 +101,18 @@ class ThreadController extends BaseController {
   final CachingManager _cachingManager;
 
   final listEmailDrag = <PresentationEmail>[].obs;
-
+  bool shiftSelectedMode = false;
```

Comment:
IMO, if it is public, it is only to get. How to do it with obs variable, public for get, and private for set?